libs/sdk-ui-filters/src/AttributeFilter/utils.ts

```diff
@@ -131,10 +131,15 @@ export function validateAttributeFilterProps(props: IAttributeFilterBaseProps) {
     if (selectionMode === "single" && filter) {
```

Comment:
Do we need to nest it here? I think we can merge it to 1 condition
Again, let's add some variable aliases for improved readability, if it's more complex condition.

Reply:
fixed